### PR TITLE
[D4C] kubernetes category added. trace_point now hook_point

### PR DESCRIPTION
**Pull request by mitodrummer**
## What does this PR do?

- Kubernetes added as a category.
- trace_point -> hook_point
- logs-cloud_defend* dataview added

## Checklist

- [x] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [x] I have verified that all data streams collect metrics or logs.
- [x] I have added an entry to my package's `changelog.yml` file.
- [x] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

### CODE REVIEW

File: packages/cloud_defend/docs/README.md [hunk 1]
```diff
@@ -174,7 +174,7 @@ responses:
 | cloud_defend.matched_selectors | ['interactiveSessions'] |
 | cloud_defend.package_policy_id | '4c9cbba0-c812-11ed-a8dd-91ec403e4f03' |
 | cloud_defend.package_policy_revision | 2 |
-| cloud_defend.trace_point | ... |
+| cloud_defend.hook_point | ['tracepoint_sched_process_fork','tracepoint_sched_process_exec'] |
```
Comment (rhysre) [inline packages/cloud_defend/docs/README.md:177]: Minor thing, in my PR, the hook points are going to be formatted with two underscores separating the type and the name, e.g. `tracepoint__sched_process_exec`, `kprobe__taskstats_exit`. See [here](https://github.com/elastic/cloud-defend/issues/111#issuecomment-1477076054)

**Comment by mitodrummer** [comment]
fleet/agent issues in latest 8.8.0 are causing CI to fail.

**Review by norrietaylor — APPROVED** [review]

**Pull request merged**